CHANGELOG_UNRELEASED.md

```diff
@@ -19,3 +19,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Infrastructure
 
 ### Misc
+- Added fixpoint and cofixpoint constructions to `finset`: `fixset`,
+  `cofixset` and `fix_order`, with a few theorems about them
+
+- Added library `finmap` which contains finite sets on a `choiceType`,
+  finite maps and finitely supported functions with domain in a
+  `choiceType` and codomain in an `eqType`.
+  See the header of `ssreflect/finmap.v` for a detailed documentation.
+
+### Changed
+
+- `eqVneq` lemma is changed from `{x = y} + {x != y}` to
+  `eq_xor_neq x y (y == x) (x == y)`, on which a case analysis performs
+  simultaneous replacement of expressions of the form `x == y` and `y == x`
+  by `true` or `false`, while keeping the ability to use it in the way
+  it was used before.
```

Comment:
The first and last entries (`fixset` and `eqVneq`) are already a part of previous releases. Also, please relocate the second item to the Added section.